server/middleware/apiErrorMiddleware.ts

```
@@ -1,9 +1,11 @@
 import type { RequestHandler } from 'express'
 import asyncMiddleware from './asyncMiddleware'
+import logger from '../../logger'
 
 export default function apiErrorMiddleware(): RequestHandler {
   return asyncMiddleware((req, res, next) => {
-    res.locals.apiErrorCallback = (_error: Error) => {
+    res.locals.apiErrorCallback = (error: Error) => {
+      logger.error(error)
```

Avoid swallowing the error!